wandb/apis/public.py

```diff
@@ -4715,11 +4733,110 @@ def save(self):
                 ],
             },
         )
+        # Save locally modified aliases
+        self._save_alias_changes()
         return True
 
     def wait(self):
         return self
 
+    @normalize_exceptions
+    def _save_alias_changes(self):
```

Comment:
It seems like we could instead change save to call this instead so that we never call update artifact for an alias update. this would allow us to have a single source of truth for both collection types

Reply:
I removed the `aliases` section from the updateArtifact mutation called in `save()` so update artifact will never make alias updates again.